src/third-party-tags/remarketing.ts

```diff
@@ -0,0 +1,23 @@
+import { GetThirdPartyTag } from '../types';
+
+declare global {
```

Comment:
Could that be moved into `src/types.ts`? Also don't know if we could find and replace the type of google_trackConversion and google_tag_params from `any` to something more strictly typed?